rest/komand_rest/actions/get/action.py

```diff
@@ -20,14 +21,18 @@ def run(self, params={}):
 
         req_headers = Common.merge_dicts(self.connection.default_headers,
                                          headers)
-        url = requests.compat.urljoin(self.connection.base_url, route)
+        url = parse.urljoin(self.connection.base_url, route)
         response = requests.get(url, headers=req_headers,
                                 verify=self.connection.ssl_verify)
         body_object = {}
         try:
             body_object = response.json()
         except ValueError:
             """ Nothing? We don't care if it fails, that could be normal """
+        # It's possible to have a successful call with no body
+        # https://stackoverflow.com/questions/32319845/python-requests-gives-none-response-where-json-data-is-expected
+        if body_object == None:
+            body_object = {}
```

Comment:
```suggestion
            body_object = {"object": {}}
```
Should this match the output structure of a valid body_object?

Reply:
I don't think that's valid.